rllib/core/examples/simple_ppo_rl_module.py

```diff
@@ -15,14 +16,17 @@
     TorchDiagGaussian,
 )
 
+import time
+
+
 torch, nn = try_import_torch()
 
 
 def get_ppo_loss(fwd_in, fwd_out):
     # this is not exactly a ppo loss, just something to show that the
     # forward train works
-    adv = fwd_in["reward"] - fwd_out["vf"]
-    actor_loss = -(fwd_out["logp"] * adv).mean()
+    adv = fwd_in[SampleBatch.REWARDS] - fwd_out[SampleBatch.VF_PREDS]
```

Comment:
Nice!
Also, can we add a TODO here that this function will be "resolved" in the future via an RLOptimizer class (that's owned by RLTrainer, which in turn will replace Policy)? Something like this.

Reply:
done